### ISSUE TITLE
OBJLoader failes reading faces for lines, but lines without faces are discarded

### ISSUE THREAD

**Issue by hamogu** [mrdoob/three.js#9204]
A line in a OBJ file is described as a bunch of vertices. Lines then need to be be followed up by face declarations, otherwise they will be dropped here: https://github.com/mrdoob/three.js/blob/master/examples/js/loaders/OBJLoader.js#L626

However, the code that parses lines starting with "f"  (https://github.com/mrdoob/three.js/blob/master/examples/js/loaders/OBJLoader.js#L484)  does not have any branch to match lines with the pattern "f single_vertex" and throws an error.

```
Error: Unexpected face line: 'f 1'
THREE.OBJLoader.prototype.parse()
 OBJLoader.js:533
THREE.OBJLoader.prototype.load/<()
 OBJLoader.js:50
.load/<()
```
Here is a minimal example
obj file:
```
mtllib /melkor/d1/guenther/projects/threejs/obj/test3.mtl

v 10 10 10
v 20 20 20
v 0 0 0 
v 0 -51.2692 269.668

g grp1
usemtl mtl1
l 1 2 3 4 
f 1
f 2
f 3
f 4
```

mtl file (but the problem is not related to any specific material)
```
newmtl mtl1
Ka 1 .5 1
Kd 1 .5 1
Ks 1 .5 1
Ns 1
Tr 0.4 illum 3
```

Possibly related to #8443 
CC: @jonnenauha @birgersp

**Comment by jonnenauha** [comment]
The `l 1 2 3 4` is already adding vertices to the geometry from https://github.com/mrdoob/three.js/blob/master/examples/js/loaders/OBJLoader.js#L377 to https://github.com/mrdoob/three.js/blob/master/examples/js/loaders/OBJLoader.js#L254

So the face `f` declarations are not needed. If you want to mix both lines and normal geometry in the same object. This will actually not work at the moment, I've seen some files that have this, but its not supported by the parser, it should track line geom and normal geom separately. You can however mix both lines and normal geom inside a obj but they need to be in separate object/group `o/g` sections.

I think your file should work as expected if you remove the `f` lines. Did you make this file by hand or was it produced by an exporter?

**Comment by hamogu** [comment]
The file was made with an exporter from the Mayavi python package, but in my real application I have thousands of lines and hundreds of objects with faces, thus I edited it down to make a minimal example. I verified that the exporter-written obj and the edited obj fail at the same line with the same error message; the `f` lines for each vertex of the line are definitely in the exporter-written file.

So, I suggest to conclude that the problem is in the exporter of Mayavi and not in the importer of three.js and thus I will close this issue. 
(For my specific case, I've found a different solution to  move the data from my python application into three.js for display).
